draft-ietf-tls-tls13.md

```diff
@@ -3695,6 +3705,16 @@ max_early_data_size
   depend on being able to send large quantities of padding in early data records.
 {:br }
 
+The PSK associated with the ticket is computed as:
+
+~~~~
+    HKDF-Expand-Label(resumption_master_secret,
```

Comment:
I forget if we already have cases where we HKDF-Expand something that's already HKDF-Expand output instead of HKDF-Extract output; we've gone back and forth on that, IIRC.